src/ec2/utils.ts

```diff
@@ -3,15 +3,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { AsyncCollection } from '../shared/utilities/asyncCollection'
-import { Ec2Client, Ec2Instance } from '../shared/clients/ec2Client'
+import { Ec2Client } from '../shared/clients/ec2Client'
 
 export interface Ec2Selection {
     instanceId: string
     region: string
 }
 
-export async function getInstancesFromRegion(regionCode: string): Promise<AsyncCollection<Ec2Instance>> {
-    const client = new Ec2Client(regionCode)
-    return await client.getInstances()
+export async function isEc2SelectionRunning(selection: Ec2Selection): Promise<boolean> {
+    const client = new Ec2Client(selection.region)
+    return await client.isInstanceRunning(selection.instanceId)
```

Comment:
why is `isEc2SelectionRunning` needed (or util.ts)?

Reply:
good catch. I think this is outdated. The `util.ts` was originally for avoiding some circular imports with the `Ec2Selection`, but I moved it to `prompter.ts` which should also solve that problem.